run_dir/static/js/index.js

```diff
@@ -118,7 +118,103 @@ $('body').on('click', '.group', function(event) {
   $($("#prio_projs_table").DataTable().column(0).header()).trigger("click")
 });
 
+function plot_sum_data(){
+  $.getJSON("/api/v1/sensorpush", function(data) {
+    var frig_series = [];
+    var freez_series = [];
+    $.each(data, function(id, sensordata){
+      var timedata = sensordata.samples;
+      var sensname = sensordata.sensor_name;
+      var sercolor = '';
```

Comment:
This doesn't seem to be used, can be removed?

Reply:
True! It's from something I tried before :-)